OPERATOR.rst

```diff
@@ -252,6 +252,8 @@ We promote at 3pm to give a cushion of time in case anything goes wrong.
 
 To do a promotion:
 
+#. Create a new Github issue with the title ``Promotion yyyy-mm-dd``
+
 #. Announce in the `#team-boardwalk Slack channel`_ that you plan to promote to ``prod``
 
 #. Make sure your ``develop`` and ``prod`` branches are up to date.
```

Comment:
```suggestion
#. Make sure your ``develop`` and ``prod`` branches are up to date. Then run
```

Reply:
For PL: there are no commands under this section, should the text section below this section be moved?

Reply:
From PL: move to the empty block